addons/medical_status/functions/fnc_addInventoryActions.sqf

```diff
@@ -46,5 +46,6 @@ _unit addAction ["OpenBag", {
     {!lockedInventory _backpackContainer} &&
     {maxLoad _backpackContainer > 0} &&
     {getNumber (_backpackConfig >> "disableInventory") != 1} &&
+    {(typeOf _target) != QEGVAR(dragging,clone)} &&
```

Comment:
While dragging, you could access the backpack of the clone
action was being added by this because

`alive x = true`
`lifeState x = "DEAD"`

Reply:
Would `isKindOf` instead of `typeOf` make more sense? Or is it not worth it?

Reply:
```suggestion
    {!(_target isKindOf QEGVAR(dragging,clone)} &&
```
this might be better if we add other mimics?
but I think typeOf is a tiny bit faster